src/main/java/compass/CompassUtilities.java

```diff
@@ -4552,11 +4564,11 @@ else if (subContext.startsWith("TABLE ")) {
 			else
 				objTypeIssueCount.put(type, objTypeIssueCount.getOrDefault(type, 0) + 1);
 		}
-		for (String t : objTypeIssueMap.keySet()) {
-			Integer iX = objTypeIssueCount.getOrDefault(t,0);
-			Integer i0 = objTypeNoIssueCount.getOrDefault(t,0);
-			//appOutput(thisProc()+"t=["+t+"] iX=["+iX+"] i0=["+i0+"] ");
-		}
+//		for (String t : objTypeIssueMap.keySet()) {
+//			Integer iX = objTypeIssueCount.getOrDefault(t,0);
+//			Integer i0 = objTypeNoIssueCount.getOrDefault(t,0);
+//			appOutput(thisProc()+"t=["+t+"] iX=["+iX+"] i0=["+i0+"] ");
+//		}
```

Comment:
The same applies for this: commented-out code is from the time before there was version control.

Reply:
this code is used for debugging. I will add a comment clarifying that.
but more generally, there is a lot of scope for code cleanup and overall rationalization in Babelfish Compass. This is just one case. That would be a sub-project by itself, and currently that does not have priority.